tools/dockerfile/test/cxx_jessie_x64/Dockerfile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM debian:jessie
+FROM gcr.io/cloud-marketplace/google/rbe-debian8@sha256:496193842f61c9494be68bd624e47c74d706cabf19a693c4653ffe96a97e43e3
```

Comment:
as discussed, let's introduce a separate rbe clang image (cxx_sanitizers..  )  and use cxx_jessie_x64 (after removing clang) for running basic C/C++ suites.

Reply:
https://github.com/grpc/grpc/blob/master/tools/run_tests/run_tests.py#L526

We actually use the current Jessie Dockerfile for clang3.5. clang3.5 is installed in https://github.com/grpc/grpc/blob/master/tools/dockerfile/test/cxx_jessie_x64/Dockerfile#L73, so we at least have to keep that version of clang, but I'm fine with removing the version installed from source.

Before updating my PR, since I forgot to remove clang, are you ok with the name `cxx_rbe_jessie_x64` or do you think `cxx_sanitizers_x64` is better?

Reply:
I think cxx_sanitizers_... is better because it's more readable.  no one knows what rbe is.